styleguide.config.js

```diff
@@ -31,75 +31,84 @@ module.exports = {
-        },
-        {
-          name: 'Headers',
-          components: () => [ 'src/molecules/PriceDiscountHeader/index.js' ]
-        },
-        {
-          name: 'Lists',
-          components: 'src/molecules/lists/**/index.js'
+          name: 'Atoms',
+          components: 'src/atoms/**/index.js'
         },
         {
-          name: 'Modal',
-          components: 'src/molecules/Modal/index.js'
+          name: 'Molecules',
+          sections: [
+            {
+              name: 'Asides',
+              components: 'src/molecules/asides/**/index.js'
+            },
+            {
+              name: 'DataRow',
+              components: 'src/molecules/DataRow/index.js'
+            },
+            {
+              name: 'FlashMessage',
+              components: 'src/molecules/FlashMessage/index.js'
+            },
+            {
+              name: 'FormFields',
+              components: 'src/molecules/formfields/**/index.js'
+            },
+            {
+              name: 'Headers',
+              components: () => [ 'src/molecules/PriceDiscountHeader/index.js' ]
+            },
+            {
+              name: 'Lists',
+              components: 'src/molecules/lists/**/index.js'
+            },
+            {
+              name: 'Modal',
+              components: 'src/molecules/Modal/index.js'
+            },
+            {
+              name: 'TextLockups',
+              components: 'src/molecules/TextLockup/index.js'
+            },
+            {
+              name: 'Other',
+              components: () => [ 'src/molecules/StepProgress/index.js' ]
+            }
+          ]
         },
         {
-          name: 'TextLockups',
-          components: 'src/molecules/TextLockup/index.js'
+          name: 'Organisms',
+          sections: [
+            {
+              name: 'Cards',
+              components: 'src/organisms/cards/**/index.js'
+            },
+            {
+              name: 'Forms',
+              components: 'src/organisms/forms/**/index.js'
+            },
+            {
+              name: 'Tables',
+              components: 'src/organisms/tables/**/index.js'
+            }
+          ]
         },
         {
-          name: 'Other',
-          components: () => [ 'src/molecules/StepProgress/index.js' ]
+          name: 'Templates',
+          components: 'src/templates/**/index.js',
         }
       ]
     },
     {
-      name: 'Organisms',
-      sections: [
-        {
-          name: 'Cards',
-          components: 'src/organisms/cards/**/index.js'
-        },
-        {
-          name: 'Forms',
-          components: 'src/organisms/forms/**/index.js'
-        },
-        {
-          name: 'Tables',
-          components: 'src/organisms/tables/**/index.js'
-        }
-      ]
-    },
-    {
-      name: 'Templates',
-      components: 'src/templates/**/index.js',
-    },
+      name: 'Design',
+      components: 'src/atoms/**/index.js'
```

Comment:
@jmcolella Yes. We don't know how they'll want to organize everything, so I just left it. It's also looking like you can't include components twice, so we'll have to figure something out.